content/maintainers-guide/processes/release/autoupgrade.md

```diff
@@ -1,27 +1,46 @@
 ---
-title: How to release a new AutoUpgrade version
+title: How to release a new Update Assistant version
 ---
 
-# Release a new version of [AutoUpgrade](https://github.com/PrestaShop/autoupgrade/) module
+# Release a new version of Update Assistant
 
-The AutoUpgrade module is released following [the same process as the native modules][native-module-release].
+New versions of Update Assistant ([`autoupgrade` module](https://github.com/PrestaShop/autoupgrade/)) are released following [the same process as the native modules][native-module-release].
 
-However, after the release has been published, a last step is necessary.
+However, after the release has been published, more steps can be necessary.
 
 ## Mark the new version as latest
 
-The AutoUpgrade module is able to detect whether a new version is available. If it detects a new version is available it will suggest to user to upgrade itself to the new higher version.
+Update Assistant can determine whether a newer version is recommended. When the current version is older than the recommended one, the module suggests upgrading to the latest version. Given the critical nature of the process, this check ensures updates are performed with all the latest improvements and bug fixes.
 
-For example if user is using module 4.16.1 and it detects that version 4.16.3 has been published, the module will inform the user that he is using an outdated version and should perform a module upgrade to version 4.16.3 .
+For example, if a user is using version 4.16.1 and version 4.16.3 has been released, the module will inform the user that they are using an outdated version and should update.
 
-The detection mechanism is a simple verification of the file https://api.prestashop.com/xml/channel.xml .
+<img src="../../../images/autoupgrade-update-needed.png" alt="Update of the module needed">
 
-Consequently when a new version of the AutoUpgrade module is published, the file https://api.prestashop.com/xml/channel.xml must be modified. It is stored in the PrestaShop API repository.
+This detection mechanism is based on the contents of the [`/autoupgrade`](https://api.prestashop-project.org/autoupgrade) endpoint of the Distribution API.
 
-{{% notice warning %}}
-**This step requires special rights.**
+The file maps ranges of PrestaShop versions to a recommended version of Update Assistant. This allows the module to:
+* Warn users when their current version is outdated,
+* Ensure updates are only allowed if the PrestaShop version falls within one of the defined ranges.
 
-Ask a maintainer from the PrestaShop Company with administrative rights on the PrestaShop API repository to perform this step.
-{{% /notice %}}
+To enable this check, update the contents of the closest exising version entry. For instance:
```

Comment:
Update where? As a PR in the dist api repo? if needed, you can redirect people to external resources and say that special rights are needed

Reply:
Good catch. I linked the file once in production but forgot the related one in GitHub

Reply:
However, I'm not sure adding the mention of special right is necessary, because everyone can submit a pull-request from now. With the old API, you need a VPN + additional Git accesses.